interface/wx/toolbar.h

```diff
@@ -159,6 +159,19 @@ class wxToolBarToolBase : public wxObject
 
     void SetDropdownMenu(wxMenu *menu);
     wxMenu *GetDropdownMenu() const;
+
+    /**
+     Mark the tool to be not visible.
```

Comment:
This is pretty misleading for a function called like this. You should say it may be used to mark the tool as being not currently visible if `@c available is @false` or something like that.